recipes/asn2gb/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
     sha256: 4b6ac3831061bc8f8c8adb011fe58c8741402a9c21c95af395aa98154c9ab775 # [linux]
 
 build:
-    number: 0
+    number: 1
```

Comment:
pinning on run-only dep can be removed (needs https://github.com/bioconda/bioconda-utils/pull/264 to make linter happy)